tests/helpers.py

```diff
@@ -1165,9 +1165,13 @@ def delete_deploymentconfig(pod_obj):
     Args:
          pod_obj (object): Pod object
```

Comment:
I think the function name should be changed to delete_deploymentconfig_pods()

Reply:
yes make sense will update in next commit

Reply:
done